src/pint/models/model_builder.py

```diff
@@ -134,6 +137,16 @@ def __call__(
             converted to TDB upon read. If "raw", an unconverted malformed TCB
             TimingModel object will be returned.
 
+        allow_T2 : True, False, or str, optional
+            Whether to convert a T2 binary model to an appropriate underlying
+            binary model. Default is False, and will throw an error upon
+            encountering the T2 binary model. If True, the binary model will be
+            converted to the most appropriate PINT-compatible binary model.
+
+        force_binary_model : str, optional
+            When set to some binary model, this will override the binary model
+            set in the parfile. Defaults to None
```

Comment:
Please give an example. Like, should it be "ELL1" or "BinaryELL1"?

Reply:
Thinking about it, what would be preferred? I'm using it as "ELL1" rather than "BinaryELL1", but how do we want this in PINT?

Reply:
I personally prefer "ELL1" just to reduce typing.

Reply:
Done